.github/workflows/push-lacchain-environment.yaml

```diff
@@ -33,17 +33,17 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
           # webapp
           PORT: '80'
-          REACT_APP_TITLE: 'LACChain Network Dashboard'
+          REACT_APP_TITLE: 'LACChain EOSIO Network Dashboard'
           REACT_APP_DEFAULT_PRODUCER_LOGO: 'https://latamlink.io/images/latamlink_logo-header.svg'
-          REACT_APP_FOOTER_LINKS: '[{"text":"LACChain Website","src":"https://www.lacchain.net"},{"text":"Block Explorer","src":"https://explorer.latamlink.io"},{"text":"Developer Guidelines","src":"https://guide.eoscostarica.io"},{"text":"Documentation","src":"https://latamlink.io/docs/eosio"},{"text":"GitHub","src":"https://github.com/eoscostarica/eosio-dashboard"}]'
+          REACT_APP_FOOTER_LINKS: '[{"text":"LACChain Website","src":"https://www.lacchain.net"},{"text":"Block Explorer","src":"https://explorer.lacchain.net"},{"text":"Developer Guidelines","src":"https://guide.eoscostarica.io"},{"text":"Documentation","src":"https://latamlink.io/docs/eosio"},{"text":"GitHub","src":"https://github.com/eoscostarica/eosio-dashboard"}]'
```

Comment:
@xavier506 thats block explorer is the Besus's one.

Reply:
good catch